tgui/packages/tgui/interfaces/CrewConsole.js

```diff
@@ -140,13 +154,64 @@ export const CrewConsoleContent = (props, context) => {
                   ({!originalTitles ? sensor.assignment_title : sensor.assignment})
                 </Table.Cell>
                 <Table.Cell collapsing textAlign="center">
-                  <ColorBox
-                    color={healthToColor( // yogs -- show death when dead
-                      sensor.oxydam,
-                      sensor.toxdam,
-                      sensor.burndam,
-                      sensor.brutedam,
-                      sensor.life_status)} />
+                  {sensor.is_irradiated ? <Icon name="radiation" color="#f0e21d" size={1} /> : ""}
+                  {sensor.is_husked ? <Icon name="ribbon" color="#ad1c09" size={1} /> : ""}
+                  {sensor.is_onfire ? <Icon name="fire" color="#f24f0f" size={1} /> : ""}
+                  {sensor.is_wounded ? <Icon name="star-of-life" color="#f50505" size={1} /> : ""}
+                </Table.Cell>
+                <Table.Cell collapsing textAlign="center">
+                  {sensor.is_robot ? (<Icon name="robot" color="#2e46cc" size={1} />
```

Comment:
holy shit my eyes. you could pull the `species.name` and have a mapping to color such as

```
ipc = "#FFFFFF",
felinid = "#000000"

Reply:
didnt know about that.. oh well